include/open62541/architecture_definitions.h

```diff
@@ -480,7 +480,7 @@ static UA_INLINE size_t
 UA_atomic_addSize(volatile size_t *addr, size_t increase) {
 #if UA_MULTITHREADING >= 100
 #ifdef _MSC_VER /* Visual Studio */
-    return InterlockedExchangeAdd(addr, increase) + increase;
+    return InterlockedExchangeAdd((LONG*) addr, (LONG) increase) + increase;
```

Comment:
VS here uses
```c++
return InterlockedExchangeAdd((volatile unsigned long *) addr, (unsigned long) increase) + increase;
```

Reply:
I get an error:  warning C4133: 'function': incompatible types - from 'volatile size_t *' to 'volatile LONG *'